### PR TITLE
Story/io 177/project status

### PR DESCRIPTION
**Pull request by EngbergJonas**
- Project card basics form status section
- RadioCheckboxField component
- Tests for new values, lists and RadioCheckboxField

### CODE REVIEW

File: src/components/shared/RadioCheckboxField.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+              />
+            ))}
+          </SelectionGroup>
+        </div>
+      )}
+    />
+  );
+};
+
+export default RadioCheckboxField;
```
Comment (kgunga) [inline src/components/shared/RadioCheckboxField.tsx:50]: wrap with memo

File: src/components/shared/RadioCheckboxField.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+interface IRadioCheckboxFieldProps {
+  name: string;
+  label: string;
+  control: HookFormControlType;
+  rules?: HookFormRulesType;
+  readOnly?: boolean;
+}
+
+const RadioCheckboxField: FC<IRadioCheckboxFieldProps> = ({ name, label, control, rules }) => {
+  const optionToBoolean = (e: ChangeEvent<HTMLInputElement>) => e.target.value === 'yes';
```
Comment (kgunga) [inline src/components/shared/RadioCheckboxField.tsx:16]: wrap with useCallback

File: src/components/shared/RadioCheckboxField.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  const valueToString = (val: boolean) => (val ? 'yes' : 'no');
```
Comment (kgunga) [inline src/components/shared/RadioCheckboxField.tsx:17]: wrap with useCallback

**Review by kgunga — CHANGES_REQUESTED** [review]

File: src/components/shared/RadioCheckboxField.tsx (hunk 1 above)
Comment (EngbergJonas) [reply src/components/shared/RadioCheckboxField.tsx]: resolved

File: src/components/shared/RadioCheckboxField.tsx (hunk 2 above)
Comment (EngbergJonas) [reply src/components/shared/RadioCheckboxField.tsx]: resolved

File: src/components/shared/RadioCheckboxField.tsx (hunk 3 above)
Comment (EngbergJonas) [reply src/components/shared/RadioCheckboxField.tsx]: resolved

**Review by kgunga — APPROVED** [review]
👍

**Pull request merged**